Makefile

```diff
@@ -72,7 +72,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0 ;\
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0 ;\
```

Comment:
can we drop the line above then?

Reply:
also 0.5.0 doesn't work for me on  go1.18.3 darwin/arm64, while 0.6.2 does and it seems to still support the needed legacy features.